test/unit/lib/helpers/detainee-attributes.test.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var Detainee = require('../../../../api/lib/helpers/detainee-attributes');
+
+describe('api/lib/helpers/detainee-attributes', () => {
+  var attrs;
+  beforeEach(() => {
+    attrs = {
+      person_id: 1234,
+      cid_id: 9990,
+      centre: 'foo',
+      gender: 'female'
+    };
+  });
+  describe('new', () => {
```

Comment:
either describe the method you're testing or the behaviour, `new` doesn't help me parse what is in test